afk/afk.go.tmpl

```diff
@@ -61,7 +62,7 @@
             {{ if gt .ExpiresAt.Unix 0 }} {{ $eta = humanizeDurationSeconds (.ExpiresAt.Sub currentTime) | printf "*%s will be back in around %s.*" $user.Username }} {{ end }}
             {{ sendMessage nil (cembed
                 "author" (sdict "name" (printf "%s is AFK" $user.String) "icon_url" ($user.AvatarURL "256"))
-                "description" (joinStr "\n\n" $eta .Value)
+                "description" (print $eta "\n\nReason: **" .Value "**")
```

Comment:
Honestly not a super big fan of this change, but if others like it better than the original, that's okay.
However, I don't think we should be providing a default message unless strictly necessary. Here's how I would format the description:

* If there is both an ETA and reason, prefer `<ETA>\n<REASON>`
* If there is only an ETA, prefer `<ETA>`
* If there is only a reason, prefer `<REASON>`
* If there is neither an ETA nor a reason, prefer `*No reason provided*`

Reply:
Fair enough, couldn't made any sense out of the abbrevation "ETA", thus not commenting on this change.